cli/command/system/version.go

```diff
@@ -140,14 +146,17 @@ func runVersion(dockerCli command.Cli, opts *versionOptions) error {
 			Os:                runtime.GOOS,
 			Arch:              runtime.GOARCH,
 			Experimental:      dockerCli.ClientInfo().HasExperimental,
-			Orchestrator:      string(dockerCli.ClientInfo().Orchestrator),
+			StackOrchestrator: string(orchestrator),
 		},
 	}
 
 	sv, err := dockerCli.Client().ServerVersion(context.Background())
 	if err == nil {
 		vd.Server = &sv
-		kubeVersion := getKubernetesVersion(dockerCli, opts.kubeConfig)
+		var kubeVersion *kubernetesVersion
+		if orchestrator.HasKubernetes() {
+			kubeVersion = getKubernetesVersion(opts.kubeConfig)
```

Comment:
nit: or we could pass `orchestrator` to `getKubernetesVersion` instead of `kubeConfig`

Reply:
Why instead of? I think getKubernetesVersion needs the kubeConfig to instantiate the kube client.

Reply:
Sorry got mixed up, I meant instead of the `dockerCli` which was passed before.